intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -209,6 +209,32 @@ \section{Defining and submitting your job}
 problem that prevented a successful run. The standard output file will contain
 the results of your calculation (here, the output of the perl script)
 
+\ifgent
+\subsection{When will my job start?}
+\label{subsec:priority}
+In practice it's impossible to predict when your job(s) will start,
+since most currently running jobs will finish before their requested walltime
+expires, new jobs by may be submitted by other users that are assigned a higher
+priority than your job(s).
+
+The clusters use a fair-share sheduling policy. There is no guarantee on when a
+job will start, since it depends on a number of factors. One of these factors is
+the priority of the job, which is determined by
+\begin{itemize}
+    \item historical use: the aim is to balance usage over users, so
+        infrequent users get a highter priority
+
+    \item requested resources (amount of cores, walltime, memory, ...): the larger
+        the resource request is, the lower the priority will be
```

Comment:
Actually, that's not fully true, we assign higher priority on some clusters for multi-node jobs for example...

The requested resources do have an impact on priority though, but we shouldn't state that requesting more resources implies lower priority, that's incorrect.

In addition, requesting less resources (e.g. not requesting all cores in a workernode) may be beneficial, since the scheduler often finds a 'gap' in the cluster more easily then (this is separate from priority...).